weasyl/character.py

```diff
@@ -359,10 +406,16 @@ def remove(userid, charid):
     return ownerid
 
 
-def fake_media_items(charid, userid, login, settings):
-    submission_url = define.cdnify_url(
-        define.url_make(charid, "char/submit", query=[userid, settings], file_prefix=login))
-    cover_url = define.cdnify_url(define.url_make(charid, "char/cover", query=[settings], file_prefix=login))
+def fake_media_items(charid, userid, login, settings, absolutify=False):
+    submission_url = define.cdnify_url(define.url_make(charid, "char/submit", file_prefix=login))
```

Comment:
I've fixed the settings param in my local copy which I haven't committed yet. 

Using the orm also breaks a few other things with settings (in views, mostly). I think for the purpose of this PR, it'd be more within scope to revert to the original query and work on that change at a later point.